frontend/src/workspace/buildingTools/ValNode.js

```diff
@@ -252,8 +252,8 @@ function ValNode(props) {
               id="form#"
               style={{
                 position: "absolute",
-                left: props.x + valueWidth * 1/3,
-                top: props.y + valueWidth * 4/3,
+                left: props.x + 25 + props.formOffsetX,
```

Comment:
why numbers and not ratios?

Reply:
Admittedly, not sure. May have been another instance of me noodling around to fix the display for my computer and not accounting for it when committing. The important bit is the addition of props.formOffsetX